tests/LinkTest.php

```diff
@@ -6,10 +6,76 @@ class LinkTest extends CopilotTagTest
 {
     public function expectedWrites()
     {
+        $assetMarkdown = "\n\n[#image: /photos/123ID]|||some caption|||\n";
+
         return [
             [
-                new Link("Hello world!"),
-                "Hello world!"
+                new Link(),
+                ""
+            ],
+            [
+                new Link(""),
+                ""
+            ],
+            [
+                new Link("Hello world!", "http://li.nk"),
+                "[Hello world!](http://li.nk)"
+            ],
+            [
+                new Link("Hello world!", "http://li.nk", Array("target" => "_blank")),
+                "[Hello world!](http://li.nk){: target=\"_blank\" }"
+            ],
+            [
+                new Link("Hello world!", "http://li.nk", Array("target" => "_blank", "title" => "rawr")),
+                "[Hello world!](http://li.nk){: target=\"_blank\" title=\"rawr\" }"
+            ],
+            [
+                new Link("Hello world!", null, Array("target" => "_blank", "title" => "rawr")),
+                "[Hello world!](){: target=\"_blank\" title=\"rawr\" }"
+            ],
+            [
+                new Link("Hello world!\n\n", "http://li.nk"),
+                "[Hello world!](http://li.nk)\n\n"
+            ],
+            [
+                new Link("  Hello world!\n\n", "http://li.nk"),
+                "  [Hello world!](http://li.nk)\n\n"
+            ],
+            [
+                new Link("  Hello world!\n\n", "http://li.nk"),
+                "  [Hello world!](http://li.nk)\n\n"
```

Comment:
What's the difference between this and the test above it?

Reply:
Check's if people are really reading the PRs ;)